r/R/dataset-write.R

```diff
@@ -133,6 +134,7 @@ write_dataset <- function(dataset,
                           max_rows_per_group = bitwShiftL(1, 20),
                           ...) {
   format <- match.arg(format)
+  format <- ifelse(as.character(format) %in% c("feather", "ipc"), "arrow", as.character(format))
```

Comment:
Just a stylistic thing, IMO this is more readable

```suggestion
  if (format %in% c("feather", "ipc")) {
    format <- "arrow"
  }
```